trade-finance-manager-ui/templates/utilisation-reports/_macros/search-reports-table.njk

```diff
@@ -0,0 +1,52 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% import "./report-reconciliation-status.njk" as reconciliationStatus %}
+
+{% macro render(params) %}
+  {% set user = params.user %}
+  {% set reports = params.reports %}
+  {% set isTfmPaymentReconciliationFeatureFlagEnabled = params.isTfmPaymentReconciliationFeatureFlagEnabled %}
+  {% set reportRows = [] %}
+
+  {% for report in reports %}
+    {% set reportPeriodHtml =
+      '<p class="govuk-body govuk-!-margin-0">' + report.formattedReportPeriod + '</p>' 
+      if not isTfmPaymentReconciliationFeatureFlagEnabled 
+      else '<a class="govuk-link govuk-link--no-visited-state" href="/utilisation-reports/' + report.reportId + '">' + report.formattedReportPeriod + '</a>'
+    %}
+
+    {{- reportRows.push([
+      {html: reportPeriodHtml }, 
+      {text: reconciliationStatus.render({
+        status: report.status,
+        displayStatus: report.displayStatus
+      }) }, 
+      {text: report.formattedDateUploaded},
+      {text: report.totalFeesReported},
+      {text: report.reportedFeesLeftToReconcile}
+    ]) | reject() }}
```

Comment:
This `| reject()` feels a bit of a hacky way to not display the output of the `push()` method but couldn't get it to work without it, any suggestions welcome

Reply:
This was where I got the ` | reject()` from, just missed this example somehow